markdown/events/2022/bytesize-bactopia.md

```diff
@@ -26,3 +26,82 @@ It is our hope that these talks / videos will build an archive of training mater
+(host) Thank you very much. I have enable now for everyone to unmute themselves, if they have questions. Otherwise we can start with the one question that is already in the chat, which is from Olaitan. He is asking: "it seems like bactopia is not in the APT repository, could you work on including it?"
+(answer) I don't know much about including tools in the apt repository. It is available from bioconda, so you can "conda install" bactopia. I think there would be many components of bactopia, that aren't in the APT repository, so I don't know how that would work. It would be something you would have to add all the dependencies to the APT repository, and I think the time required for that and the learning, I don't have the bandwidth for at the moment. Definitely consider using the bioconda install and then from there you can use conda docker singularity.
+
+[17:55](https://www.youtube.com/watch?v=egjgcmeJ0wQ&t=1075)
+(host) okay thank you. Are there any more questions from the audience?
+
+(question) I do have a follow-up question. It's not really based on the APT repository thing, it's the the workflow. In terms of the different steps, that bactopia does. I didn't get what Robert meant by the final step that he talked about. Something about the analysis. I was wondering what are the specific things, like what are you measuring at the end of the day with bactopia? Specifically in terms of the omics analysis.
+(answer) It's going to include pretty much all your standard bacterial genomics. You're going to qc the reads, how well did you sequence your sample, what's the average read length, all that fun stuff. Then it's going to characterize your sample. what nst schema? Does it have certain antibiotic resistance genes that you may be interested in? Does it have snips and endels against a reference genome that you selected? How does it compare to public available genomes, does it look like what you expected? If you thought you sequenced the staph aureus, and you know it came up as looking more like enterococcus, that's something. Those are the type of analysis results. On the bactopia documentation there's an overview of the workflow at each step and then output overviews on all the output files that you get for both bactopia and all the bactopia tools. Those output give you a description of what's in the each of the files.
+
+[19:52](https://www.youtube.com/watch?v=egjgcmeJ0wQ&t=1192)
+(question) Okay thanks! My final question is, I saw you integrated this with illumina reads and I believe nanopore reads. What happened to pacbio?
+(answer) Honestly I just haven't been exposed to pacbio data much so far in my analyses and my studies. I think if I start using pacbio, then pacbio data will come in, otherwise I think I would need support from the community to add that type. Just because I don't have the opportunity to use it on a daily basis like I do illumina and nanopore. We need someone else to help out there.
```

Comment:
```suggestion
(answer) Honestly I just haven't been exposed to PacBio data much so far in my analyses and my studies. I think if I start using PacBio, then PacBio data will come in, otherwise I think I would need support from the community to add that type. Just because I don't have the opportunity to use it on a daily basis like I do Illumina and Nanopore. We need someone else to help out there.
```